modules/component/types/pickChildrenType.ts

```diff
@@ -0,0 +1,8 @@
+import {PPP} from './pickValueFromComponent'
+
+/**
+ * Pick component children type from component or component props
```

Comment:
remove the first `component`. Can just be `children type from Comp or Comp props`